app/helpers/feedback_helper.rb

```diff
@@ -0,0 +1,42 @@
+module FeedbackHelper
+  def survey_prompt(question)
+    question.prompt.strip.tap do |prompt|
+      prompt << ":" unless prompt =~ /[.?!:]$/
+      prompt << "*" if question.required?
```

Comment:
Could also be a css :after pseudoelement based on the required attribute.

Reply:
:+1: